css/styles.css

```diff
@@ -0,0 +1,115 @@
+body {
+  font-family: 'Ubuntu'; sans-serif
+  text-align: center;
+  background-color: #F2F2F2;
+  color: black;
+}
+
+img.profile {
+  float: right;
+  height: 200px;
+  width: 200px;
+  border: 2px solid #668B8B;
+  padding: 5px;
+  margin: 10px 20px 10px 10px;
+}
+
+img.experience {
+  height: 100px;
+  width: 100px;
+  border: 2px solid #668B8B;
+  padding: 5px;
+  margin: 10px 20px 10px 10px;
+}
+
+header {
+  background-color: #B4CDCD;
+  text-align: center;
```

Comment:
@sofiaayala this can be omitted since it is declared in the body and will be inherited through out the document.